methods/incidentReports.coffee

```diff
@@ -13,6 +13,7 @@ Meteor.methods
   addIncidentReport: (incident, userEventId) ->
     user = Meteor.user()
     checkPermission(user._id)
+    incident = incidentReportSchema.clean(incident)
```

Comment:
I don't think we should clean the incident here. If extra parameters are being added it would be better for it to cause a validation error so we can catch the error. Cleaning incidents before passing them to this method would be fine.